### PR TITLE
[TOOL-1417] Update privacy policy link and add message below "Manage privacy preferences" in completeOptions

### PR DESCRIPTION
**Pull request by sri-shippo**

### CODE REVIEW

File: src/cm.css [hunk 1]
```diff
@@ -131,7 +131,7 @@ a:focus {
   color: var(--text-color);
   margin: 0 0 18px 0;
   text-align: center;
-  font-family: "Rethink Sans", sans-serif;
+  font-family: "Rethink Sans", "Neue Haas Unica Bold", sans-serif;
```
Comment (zembrzuski-shippo) [inline src/cm.css:134]: Please check the website cookie banner:
image

It has font-weight 700
image

---

Look at how it looks on the webapp with font-weight 700 (too bold)
image


When changing from 700 to 500, IMO it looks better:
image

--- 

Given that said, I would recommend the following
1. using the this font config: `Rethink Sans / 700` and `neue-haas-unica / 500`
2. changing from `"Neue Haas Unica Bold"` to `neue-haas-unica` (this is the font definition in the website)

Comment (zembrzuski-shippo) [inline src/cm.css]: in the webapp, the font name is neue-haas-unica, I guess.
image

**Comment by sri-shippo** [comment]
I've updated with the changes except changing font color for 'Eseential Purposes'. I will try to check this out further.

image

**Comment by sri-shippo** [comment]
This is preview after changes:

image

**Review by zembrzuski-shippo — APPROVED** [review]
✅

**Pull request merged**